ecommerce/android/Getflix/app/src/main/java/com/example/getflix/models/CardModel.kt

```diff
@@ -2,12 +2,12 @@ package com.example.getflix.models
 
 import com.google.gson.annotations.SerializedName
 
-data class CardModel (
+data class CardModel(
 
-        @SerializedName("id") val id : Int,
-        @SerializedName("name") val name : String,
-        @SerializedName("owner_name") val owner_name : String,
-        @SerializedName("serial_number") val serial_number : Int,
-        @SerializedName("expiration_date") val expiration_date : ExpirationDateModel,
-        @SerializedName("cvc") val cvc : Int
+        @SerializedName("id") val id: Int,
+        @SerializedName("name") val name: String,
+        @SerializedName("owner_name") val owner_name: String,
+        @SerializedName("serial_number") val serial_number: Int,
+        @SerializedName("expiration_date") val expiration_date: String,
+        @SerializedName("cvc") val cvc: Int
```

Comment:
I did not notice but our all data models serialized name are totally used wrong. I will change others as well. So, we can start this one. Could you change it this to the below code?
```suggestion
        val id: Int,
        val name: String,
        @SerializedName("owner_name") val ownerName: String,
        @SerializedName("serial_number") val serialNumber: Int,
        @SerializedName("expiration_date") val expirationDate: String,
        val cvc: Int
``` 
Because this annotation can be used to serialize a field with a different name instead of an actual field name in order to follow the naming convention

Reply:
I didn't understand why it is wrong. Serialized names don't have to be different from the real variable names. I put the serialized names in case of any change in the names of the variables. So, it is not "totally wrong" and doesn't have to be updated.